packages/browser-repl/package.json

```diff
@@ -32,6 +32,7 @@
     "check": "npm run lint && npm run depcheck",
     "depcheck": "depcheck",
     "compile": "tsc -p tsconfig.json",
+    "bootstrap": "npm run compile",
```

Comment:
Missed removing this. Already amended the commit. Will push later after this evergreen run finishes.